crates/rspack_core/src/dependency/mod.rs

```diff
@@ -434,6 +445,22 @@ pub trait ModuleDependency: Dependency {
   }
 }
 
+pub trait ImportDependencyTrait: ModuleDependency {
```

Comment:
This seems unnecessary

Reply:
It's intentional. we will add `ImportWeakDependency` after now.